src/util/AddDefaultValuesProcessor.ts

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+export function addingDefaultMedicationRequestProperties(entry: any) {
+  if (!entry?.resource?.status) {
```

Comment:
Can we use `import { MedicationRequest } from "fhir/r4"` instead of  `any`?

Reply:
I don't think we can use them, I have tried it says resource is not a property of MedicationRequest. My reasoning for this would be that MedicationRequest is present inside a resource so it is not accepting entry?.resource.